build/Dockerfile

```diff
@@ -1,4 +1,71 @@
+# build stage for building binaries
+FROM registry.access.redhat.com/ubi8/go-toolset:1.12.8-32 as build
+LABEL stage=build
+
+# Build WMCB
+# WORKDIR is not resulting in creating this directory, so we are forcing it to happen. The image does not allow us to
+# create /build/, forcing us to use $HOME which /opt/app-root/src/
+RUN mkdir $HOME/build/
+WORKDIR $HOME/build/
+RUN git clone https://github.com/openshift/windows-machine-config-bootstrapper.git
+WORKDIR windows-machine-config-bootstrapper
+RUN make build
+
+# Build hybrid-overlay
+WORKDIR $HOME/build/
+RUN git clone https://github.com/openshift/ovn-kubernetes/
```

Comment:
Pinning might be a problem.

Reply:
I am not sure what you are asking for. Do you want me to pin it? I thought we would pin once we cut over to 4.6.